chapter11/rwd/ex2-11.pl

```diff
@@ -0,0 +1,13 @@
+#!/urs/bin/perl
+
+use strict;
+use warnings;
+use DateTime;
+
+my $today = DateTime->today( );
+my $out_date = $today->subtract_datetime( DateTime->new( year => $ARGV[0], month => $ARGV[1], day => $ARGV[2] ) );
+
+print $out_date->years() . " years, " . $out_date->months() . " months, and " . $out_date->days() . " days\n";
+
+exit 1;
+
```

Comment:
We discussed this last week, and why it was a bad habit that needs to be broken.